coordinator/src/services/translation/incoming/cancellation/cancellation-medication-request.ts

```diff
@@ -166,39 +166,11 @@ function createItemNumberIdentifier(pertinentInformation1: PertinentInformation1
   return [createIdentifier("https://fhir.nhs.uk/Id/prescription-order-item-number", id.toLowerCase())]
 }
 
-function getMedicationCodeableConcept() {
-  return {
-    "coding": [{
-      "system": "http://snomed.info/sct",
-      "code": "763158003",
-      "display": "Medicinal product"
-    }]
-  }
-}
-
-function getMedicationGroupIdentifier(pertinentInformation2: PertinentInformation2) {
-  const id = pertinentInformation2.pertinentPrescriptionID.value._attributes.extension
-  return createIdentifier("https://fhir.nhs.uk/Id/prescription-order-number", id)
-}
-
-function medicationRequestHasDispenser() {
-  return false
+function createMedicationCodeableConcept() {
+  return createCodeableConcept("http://snomed.info/sct", "763158003", "Medicinal product")
```

Comment:
Is there a point to this wrapper function? Could we inline it?

Reply:
I'll go one better and make a constant for it.

Reply:
Done